planetarypraxis/smartforests
# feat: add past_contributor flag to display those contributors last

## Description
Adds a checkbox to the Contributor page, `past_contributor`, and displays these contributors last on the contributors page (but within their section).

## Motivation and Context
https://linear.app/commonknowledge/issue/SF3-42/add-sections-on-contributors-page#comment-7f9616c0

## How Can It Be Tested?
Set this flag on some contributors and check they are displayed last in their section.
